src/cmd/pay.rs

```diff
@@ -322,4 +362,31 @@ mod tests {
         assert_eq!(Token::from(50000000), payee.amount);
         assert_eq!(TokenInput::Mobile, payee.token);
     }
+
+    #[test]
```

Comment:
none of these existing or new tests actually test the cmdline "one" and "multi" paths. They just test the serialization. I decided that it was better to handle that in a later PR